package.json

```diff
@@ -12,10 +12,7 @@
     "test": "nx run-many --target=test --all --parallel",
     "test:ci": "docker run -v $PWD:/cypress --net=host -w /cypress -e CYPRESS_PROJECT_ID --entrypoint=cypress cypress/included:7.5.0 run  --project .",
     "build": "cd demos/html/generic && npm install && npm start &",
-    "lint": "npm run lint-js && npm run lint-html && npm run lint-css",
```

Comment:
Even though the goal is to prettify the files, wouldn't it be interesting to have a linting command so that we can check whatever file without formatting it, yet?

Reply:
Fixed on a new commit, re-added the lint functionalities to format and lint